_source/resources.md

```diff
@@ -24,7 +24,7 @@ about the PMHC MDS. It will be updated regularly as new issues are raised by
 PMHC MDS users.
 
 ## 4. Guidance on client consent
-[The guidance on client consent](/doc/pmhc-consent-guidance-D16-1362220.pdf) provides suggested wording that PHNs may
+[The guidance on client consent](/doc/pmhc-consent-guidance-20240430.pdf) provides suggested wording that PHNs may
```

Comment:
It's worth including a date; I know its in the filename.. but lets make it explicit.  eg. add `(April 2024)` either in the link text or before "provides".